src/cli/repo-checkout.js

```diff
@@ -102,11 +108,6 @@ export default async function main(config, argv) {
   // Checkout the underlying repository before running repo...
   await checkout(config, checkoutArgs);
 
-  // Running repo sync in parallel is only safe for entirely new initializations
-  // when we can though this is up to 2x faster...
-  let canUseParallelSync =
-    !await fs.exists(fsPath.join(args.directory, '.repo'));
```

Comment:
Do we want to push this down into the `vcsRepo` object? Or was it broken logic?  (this can be done in a separate PR)